angular/projects/spark-angular/src/lib/components/sprk-accordion-item/sprk-accordion-item.component.ts

```diff
@@ -19,106 +21,145 @@ import { toggleAnimations } from '../sprk-toggle/sprk-toggle-animations';
-          class="sprk-c-Accordion__content sprk-b-TypeBodyTwo"
-        >
+        <div [id]="accordion_controls_id" [ngClass]="getContentClasses()">
           <ng-content></ng-content>
         </div>
       </div>
     </li>
   `,
-  animations: [toggleAnimations.toggleContent]
+  animations: [toggleAnimations.toggleContent],
 })
 export class SprkAccordionItemComponent implements OnInit {
+  // TODO - Remove as part of Issue 3597
   /**
-   * The value supplied will be rendered
-   * inside the title area of the Accordion item.
+   * Deprecated: use `heading` instead. The value supplied will be
+   * rendered inside the heading area of the Accordion Item.
    */
   @Input()
   title: string;
   /**
-   * The value supplied will be assigned to the
-   * `data-analytics` attribute on the component.
-   * Intended for an outside
-   * library to capture data.
+   * The value supplied will be rendered inside the heading area of the
+   * Accordion Item.
+   */
+  @Input()
+  heading: string;
+  /**
+   * The value supplied will be assigned to the `data-analytics` attribute
+   * on the component. Intended for an outside library to capture data.
    */
   @Input()
   analyticsString: string;
   /**
-   * The value supplied will be assigned
-   * to the `data-id` attribute on the
-   * component. This is intended to be
-   * used as a selector for automated
-   * tools. This value should be unique
-   * per page.
+   * The value supplied will be assigned to the `data-id` attribute on the
+   * component. This is intended to be used as a selector for automated tools.
+   * This value should be unique per page.
    */
   @Input()
   idString: string;
   /**
-   * Expects a space separated string
-   * of classes to be added to the
-   * component.
+   * Expects a space separated string of classes to be added to the component.
    */
   @Input()
   additionalClasses: string;
+  // TODO - Remove as part of Issue 3597
   /**
-   * Expects a space separated string
-   * of classes to be added to the
-   * heading in the Accordion item.
+   * Deprecated - use `headingAdditionalClasses` instead. Expects a space
+   * separated string of classes to be added to the heading in the Accordion
+   * Item.
    */
   @Input()
   additionalHeadingClasses: string;
   /**
-   * The Accordion item will use this to decide
-   * if it should be open or closed on first render.
-   * (Interacting with the toggle will override this input.)
+   * Expects a space separated string of classes to be added to the heading in
+   * the Accordion Item.
+   */
+  @Input()
+  headingAdditionalClasses: string;
+  /**
+   * The Accordion item will use this to decide if it should be open or closed
+   * on first render. (Interacting with the toggle will override this input.)
    */
   @Input()
   isOpen = false;
+  // TODO - Remove as part of Issue 3597
   /**
-   * If `true`, the active CSS class
+   * Deprecated - use additionalClasses instead. If `true`, the active class
```

Comment:
I don't think the 'use additionalClasses instead' part is relevant here. We just want to mark it deprecated since we didn't actually replace it with anything.